### PR TITLE
Update base sdk references to 2.4.0-beta1.

### PR DESCRIPTION
**Pull request by dnduffy**

### CODE REVIEW

File: src/Microsoft.ApplicationInsights.AspNetCore/project.json [hunk 1]
```diff
@@ -93,9 +93,9 @@
...
                 "System.Net.Http": "4.3.1",
-                "Microsoft.ApplicationInsights.DependencyCollector": "2.3.0-beta3",
-                "Microsoft.ApplicationInsights.PerfCounterCollector": "2.3.0-beta3",
-                "Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel": "2.3.0-beta3"
+                "Microsoft.ApplicationInsights.DependencyCollector": "2.4.0-beta1",
+                "Microsoft.ApplicationInsights.PerfCounterCollector": "2.4.0-beta1",
+                "Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel": "2.4.0-beta1"
             }
         },
         "netstandard1.6": {
```
Comment (yantang-msft) [inline src/Microsoft.ApplicationInsights.AspNetCore/project.json]: The NETStandard.Library needs to be upgrade to 1.6.1 (and test projects) if referencing AI base 2.4

Comment (dnduffy) [inline src/Microsoft.ApplicationInsights.AspNetCore/project.json]: What do we need from 1.6.1?

Comment (yantang-msft) [inline src/Microsoft.ApplicationInsights.AspNetCore/project.json]: AI base depends on 1.6.1, I don't know what it needs.

Comment (dnduffy) [inline src/Microsoft.ApplicationInsights.AspNetCore/project.json]: Updated

**Pull request merged**